### PR TITLE
sh-shell: nounset mode fixes

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

**Review by pawamoy — APPROVED** [review]
LGTM, thanks :+1: 

You have the necessary permissions to merge right? I'll leave it to you in case you want to push more commits.

**Comment by scop** [comment]
Yep, I have permissions, merged. Nothing else upcoming right now.

**Pull request merged**
